.azure-pipelines/noop-pipeline.yml

```
@@ -1,28 +1,95 @@
-trigger:
-
 pr:
   branches:
     include:
       - docs/*
       - tracer/README.md
```

I don't think we can just use this approach. For example, this pipeline will run if there are changes to the readme, even if there are also changes to the source code. The checks will then clobber the checks from the "real" pipeline. Also note that wildcards are not supported. And that you're using `branches:` not `paths:` 😉 

Instead, I think we need something like this, though this will also need testing (and we should set up the reverse in the ultimate-pipeline)

```suggestion
pr:
  paths:
    exclude:
      - .
    include:
      - blog/
      - docs/
      - tracer/README.md
      - tracer/src/Datadog.Trace/DuckTyping/README.md
      - tracer/samples
      - .github/
      - LICENSE
      - LICENSE-3rdparty.csv
      - NOTICE
      
```

---

Thanks, I think I have other issues as well as the status wasn't reported below. Looking into it